pkg/scripts/node.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scripts
+
+const (
+	drainNodeScriptTemplate = `
+kubectl drain {{ .NODE_NAME }} --ignore-daemonsets
```

Comment:
Maybe `--ignore-daemonsets` should be optional, i.e. guarded with a flag, but I don't think that's needed right now.

Reply:
ignore-daemonsets is needed, otherwise it will fail and block the whole thing. so it's OK.